tianshou/data/buffer.py

```diff
@@ -169,7 +170,7 @@ def update(self, buffer: 'ReplayBuffer') -> None:
             return
         i = begin = buffer._index % len(buffer)
         origin = buffer._get_stack_num()
-        buffer._set_stack_num(0)
+        buffer._set_stack_num(1)
```

Comment:
By the way, could you please change L259 to this style? I have no more comment.

Reply:
you mean using `_get_stack_num` ? Why just this one ?

Reply:
Exactly. Because there is just L259 a place that use self._stack instead of _get_stack_num

Reply:
Okay. Same as 208 and 209. My bad.

Reply:
Ok so I propose to use Python `@property` decorator. You can have a look to the changes.